frontend/src/views/WheresMyGene/components/HeatMap/components/XAxisChart/index.tsx

```diff
@@ -135,6 +137,9 @@ export default function XAxisChart({ geneNames }: Props): JSX.Element {
       left={Y_AXIS_CHART_WIDTH_PX + CHART_PADDING_PX}
       height={xAxisHeight}
     >
+      {!isSSR() && (
```

Comment:
Let's add a note to remove this temp fix after https://github.com/chanzuckerberg/sci-components/pull/581 is published!